lib/matplotlib/colors.py

```diff
@@ -1245,21 +1245,19 @@ def __init__(self, vmin=None, vmax=None, clip=False):
             provided, they default to the minimum and maximum values of the input,
             respectively.
 
-
         clip : bool, default: False
             Determines the behavior for mapping values outside the range
             ``[vmin, vmax]``.
 
-            If *clip* is ``False``, values outside ``[vmin, vmax]`` are also transformed
-            linearly, leading to results outside ``[0, 1]``. For a standard use with
-            colormaps, this behavior is desired because colormaps mark these outside
-            values with specific colors for over or under.
-
-            If *clip* is ``True``, values outside ``[vmin, vmax]`` are set to 0 or 1,
-            depending on which boundary they're closer to. This makes these values
-            indistinguishable from regular boundary values and can lead to
-            misinterpretation of the data.
+            If clipping is off, values outside the range ``[vmin, vmax]`` are
+            also transformed, resulting in values outside ``[0, 1]``.  This
+            behavior is usually desirable, as colormaps can mark these *over*
+            and *under* values with specific colors.
```

Comment:
Should be same order as `vmin, vmax` or `0, 1`:
```suggestion
            behavior is usually desirable, as colormaps can mark these *under*
            and *over* values with specific colors.
```

Reply:
Thanks, fixed (throughout).